galsim/chromatic.py

```diff
@@ -721,16 +735,17 @@ def shift(self, *args, **kwargs):
                     args[0](700.).x
                     # If the function returns a Position, recast it as a function returning
                     # a numpy array.
+
                     def offset_func(w):
                         d = args[0](w)
-                        return np.asarray( (d.x, d.y) )
+                        return np.asarray((d.x, d.y))
```

Comment:
Here's another case where the extra spaces add clarity.  With the spaces, it is much clearer that you are making an array from a tuple.  It's harder to notice what you are doing with the parentheses right up against each other.